src/main/java/net/kinguin/leadership/consul/Cluster.java

```diff
@@ -44,6 +44,10 @@ public boolean claimLeadership() throws JsonProcessingException {
         return (boolean) response.getValue();
     }
 
+    public ClusterConfiguration getClusterConfiguration() {
```

Comment:
Maby it should be better to rename ClusterConfiguration class to ClusterProperties - it will be more spring standard.

Reply:
Seems like good idea. I named this class ```ClusterConfiguration``` cause my first thought was setup like class for factory not just properties holder.